integrated_channels/README.md

```diff
@@ -0,0 +1,4 @@
+# Integrated Channels
+
+Integrated channels are external systems that manage Enterprise Customer data. This package contains logic
+for configuring integrated channels and sending data between us and an integrated channel.
```

Comment:
Not yet, actually. I'll do it at the end when wrapping up, because that's when we'll know 100% how everything is about to work. Otherwise I might have to come back here repeatedly to update that.

Reply:
Sounds good!